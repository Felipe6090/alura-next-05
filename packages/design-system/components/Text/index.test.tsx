import { Text } from "./index";
import {
  customRender,
  screen,
} from "@alura/test-commons/react-testing-library";

const render = customRender();

describe("<Text />", () => {
  it("renders h1", () => {
    const { container } = render(<Text tag="h1">Sample text</Text>);
    // screen.debug();
    expect(container).toMatchSnapshot();
  });
});
