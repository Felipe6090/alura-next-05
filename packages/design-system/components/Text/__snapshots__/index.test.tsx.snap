// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> renders h1 1`] = `
<div>
  <h1
    class=""
  >
    Sample text
  </h1>
</div>
`;
